### PR TITLE
add cli command to update recipient statuses, and fix recipient status updating in the status update callback

### PR DESCRIPTION
**Pull request by mrotondo**

### CODE REVIEW

File: app/models/recipient.rb [hunk 1]
```diff
@@ -53,4 +53,14 @@ class Recipient < ApplicationRecord
   scope :imported, -> { where(sms_status: :imported) }
 
   delegate :program, to: :message_batch, allow_nil: true
+
+  def update_sms_status
+    if sms_message
+      if %w(delivered).include? sms_message.status
```
Comment (jhernandez-cfa) [inline app/models/recipient.rb:59]: Do we need to check for the 'sent' status here as well?

Comment (jhernandez-cfa) [reply app/models/recipient.rb:59]: Ok, that makes sense

Comment (mpidcock) [reply app/models/recipient.rb:59]: so the status will remain `api_success` until we know if it is delivered? This means messages that are `sent` would block more messages from being queued. Probably not an issue, unless there's some kind of provider outage that fails to update the delivered status?

**Review by mpidcock — APPROVED** [review]

File: app/models/recipient.rb (hunk 1 above)
Comment (mrotondo) [reply app/models/recipient.rb:59]: Good point - we should keep an eye on this to see how long things stay in `sent` and potentially add a new Recipient status to reflect it if we find that it's blocking us

**Pull request merged**